adding_species_manuscript.md

```diff
@@ -74,7 +74,7 @@ preamble: |
 
 To include a species into the stdpopsim catalog it is required that the species has at least: 
 - (1) a **genome assembly**, with contigs being assembled at the chromosome level or nearly so. By using chromosome level information to simulate genomes,  stdpopsim indirectly simulates the effect of linked selection. [maybe discussion next text -> ] In the near future this won't be a problem for multiple species, since with (i) the advent of long-read sequencing technologies [ref] and because of (ii) initiatives such as B10K, XXXX, and G10K near XXX error-free reference genome assemblies [Rhie et al, 2021, REF, REF] spanning more than XXX vertebrate species are expected to be generated.  
-- (2) It also requires that the species has a plausible and citeable **mutation rate** estimate. Both phylogenetic mutation rate [ref], germline *de novo* mutations, or mutation rates estimates based on mutation accumulation studies [ref] are welcomed. [commment: using the mutation rate of a closely related species is not a big deal, either??? I know you guys use the drosophila one for the anopheles].
+- (2) It is also required that the species has a plausible and citeable **mutation rate** estimate. Both phylogenetic mutation rate [ref], germline *de novo* mutations, or mutation rates estimates based on mutation accumulation studies [ref] are welcomed. [commment: using the mutation rate of a closely related species is not a big deal, either??? I know you guys use the drosophila one for the anopheles].
 - (4) Since both mutation and recombination shape the genetic diversity of genomes, stdpopsim also requests a **recombination rate** estimate for the species. Ideally, a chromosome level recombination map--e.g., allowing to more precisely infer the effect of selective interference--or at least a citeable single recombination rate estimate.
```

Comment:
(3) and (4) swapped?

Reply:
```suggestion
- (4) Since both mutation and recombination shape the genetic diversity of genomes, stdpopsim also requires a **recombination rate** estimate for the species. Ideally, this should be a chromosome level recombination map, which allows more precise inference of the effect of selective interference. At minimum, it is a citeable single recombination rate estimate for the whole genome.
```